elastalert/loaders.py

```diff
@@ -606,12 +606,14 @@ def get_import_rule(self, rule):
         return expanded_imports
 
     def get_rule_file_hash(self, rule_file):
-        rule_file_hash = ''
```

Comment:
Was return `string` when file not found, which when concatenated with `bytes` in the case of import rules,

https://github.com/jertel/elastalert2/blob/caa0de3754c83c8248231e3733148bfa9536e032/elastalert/loaders.py#L613

throws `TypeError: can't concat str to bytes`